src/ontology/edits/oeo-model.omn

```diff
@@ -3013,6 +3049,78 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1711",
         OEO_00010252
     
     
+Class: OEO_00360020
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A region of relevance is a spatial region that is mentioned in any study or analysis."@en,
```

Comment:
A `study` is a `process`. We do not have yet `analysis`, but at least `sensitivity analysis` is a `process`, too. Processes do no contain (store) information but an `information content entity like `study report` can.

Therefore I suggest a modification:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A region of relevance is a spatial region that is mentioned in any study report."@en,
```

Then we can also axiomatise this as `'region of relevance' 'part of' some 'study report'`

Reply:
If we rather wanted to express that `a region of relevance` is used in a `study` or analysis, I suggest the following definition change:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A region of relevance is a spatial region that is used in a study or analysis."@en,
```

Reply:
What about
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A region of relevance is a spatial region that is mentioned  in any study report or considered/used in a study or analysis."@en,
```

Reply:
In my view that combination makes it even less clear what we really mean with a `region of relevance`. 

And it would imply the axiom `'region of relevance' ('is part of' some 'study report') or ('is used by' some study or analysis)` which is not nice.

Reply:
> If we rather wanted to express that `a region of relevance` is used in a `study` or analysis, I suggest the following definition change:

I am okay with that.